### ISSUE TITLE
Screen reader accessibility

### ISSUE THREAD

**Issue by tylerchambliss** [WilliamSchack/Spotify-Downloader#109]
### Checklist

- [x] I am using the latest version as listed [here](https://github.com/WilliamSchack/Spotify-Downloader/releases/latest)
- [x] I have verified that this feature does not already exist

### Provide a description of the feature or change

Hello William. I'm Tyler. I'm a blind person. This means that I have to use a software called a screen reader. This software speaks what my cursor is focused on as I use the computer. It's called NVDA or Nonvisual Desktop Access for short. if you want to download it and see how a blind person uses a computer. Well basically the way it knows what the name of a button or whatever is is via the labels you've set for your elements of your ui. If your UI has no labels or if it's just graphics the screen reader can't do anything with it and we're left unable to use a program. Well in short the problem here with the spotify downloader is that none of the buttons are labeled so using the interface is nearly impossible unless I use optical character recognition to half navigate. I'd really love to get some proper labels on the buttons if possible. Thank you for making this amazing little tool. I just hope you can fix screen reader access. 

### Propose a solution

Label the UI elements and research best practices for screen reader access to improve your programs

### Complete Log

```shell

```

**Comment by WilliamSchack** [comment]
I actually didnt know that the buttons needed to be setup for accessibility and screen readers didnt even cross my mind when making the ui. Looked into it turns out there is an accessible name option which i have set for all the buttons which will release with the next update. I have tested it with NVDA and it seems to be good, but if there are any more accessibility issues feel free to let me know about them and I will fix them.

I will close this issue when the next update is released, thanks for letting me know about this :)
